tests/v2/validation/nodescaling/scaling_node_driver_aks_test.go

```diff
@@ -0,0 +1,86 @@
+package nodescaling
+
+import (
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters/aks"
+	"github.com/rancher/rancher/tests/framework/extensions/scalinginput"
+	"github.com/rancher/rancher/tests/framework/pkg/config"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type AKSNodeScalingTestSuite struct {
+	suite.Suite
+	client        *rancher.Client
+	session       *session.Session
+	scalingConfig *scalinginput.Config
+}
+
+func (s *AKSNodeScalingTestSuite) TearDownSuite() {
+	s.session.Cleanup()
+}
+
+func (s *AKSNodeScalingTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	s.session = testSession
+
+	s.scalingConfig = new(scalinginput.Config)
+	config.LoadConfig(scalinginput.ConfigurationFileKey, s.scalingConfig)
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(s.T(), err)
+
+	s.client = client
+}
+
+func (s *AKSNodeScalingTestSuite) TestScalingAKSNodePools() {
+	var oneNode int64 = 1
+	var twoNodes int64 = 2
```

Comment:
This seems missing in 2.8, is it expected?

Reply:
v2.8 has them in a single file

Reply:
Oh makes sense, so 2.7 is not updated right? @markusewalker. I missed those additions on 2.8 PR

Reply:
yeah I need to update this branch, I'll do that now.